sky/provision/oci/instance.py

```diff
@@ -292,11 +293,11 @@ def open_ports(
     provider_config: Optional[Dict[str, Any]] = None,
 ) -> None:
     """Open ports for inbound traffic."""
-    # OCI ports in security groups are opened while creating the new
-    # VCN (skypilot_vcn). If user configure to use existing VCN, it is
-    # intended to let user to manage the ports instead of automatically
-    # opening ports here.
-    del cluster_name_on_cloud, ports, provider_config
+    del cluster_name_on_cloud
+    assert provider_config is not None
+    region = provider_config['region']
+    query_helper.add_ingress_rules(region,
+                                   resources_utils.port_ranges_to_set(ports))
```

Comment:
I'm a little bit curious why this does not require the cluster name. Will this open the ports for all instance in the region? Our semantic for open ports is to only open it for a specified cluster.

Reply:
Now I changed the implementation to open_ports/cleanup ports per cluster.